content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -25,7 +25,7 @@ Follow this guide to deploy Istio and connect a virtual machine to it.
 
 1. Create a virtual machine
 1. Set the environment variables `VM_APP`, `WORK_DIR` , `VM_NAMESPACE`,
-and `SERVICE_ACCOUNT`
+and `SERVICE_ACCOUNT` on your cluster
```

Comment:
These aren't on your cluster, these are on your machine that you're using to setup the cluster.

Reply:
Ok :)